CHANGELOG.md

```diff
@@ -7,6 +7,32 @@ this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 <!-- towncrier release notes start -->
 
+## 0.3.0 (2023-09-06)
+
+
+### Deprecations
+
+- Removed the `batch_work_size` configuration. Integrations should use the async generator syntax instead (PORT-4616)
+
+### Features
+
+- Added support for a configurable resync interval for integrations (PORT-4616)
+- Added a new feature that will abort a running resync if a new resync is attempting to start (PORT-4619)
+
+### Improvements
+
+- Changed the way an empty port app config is handled in the `PortAppConfig Handler` (PORT-4483)
+- Added yaml linter (#1)
+- Removed the ocean version parameter from the scaffold and getting automatically (#2)
+- Changed the publish integration workflow to get the integration version from the `pyproject.toml` file of the integration and not from the `spec.yml` file (#3)
+
+### Bug Fixes
+
+- Fixed a bug when rollbacking blueprint it would rollback all the blueprints instead only the one who failed (blueprints-to-rollback)
+- Fixed a bug when there are falsy jq evaluation so ocean would them from the entity properties (include-falsy-jq-evaluate)
+- Fixed the injections in config.yaml to be wrapped with "" (#1)
```

Comment:
```suggestion
- Added yaml linter
- Removed the ocean version parameter from the scaffold and getting automatically
- Changed the publish integration workflow to get the integration version from the `pyproject.toml` file of the integration and not from the `spec.yml` file

### Bug Fixes

- Fixed a bug when rollbacking blueprint it would rollback all the blueprints instead only the one who failed (blueprints-to-rollback)
- Fixed a bug when there are falsy jq evaluation so ocean would them from the entity properties (include-falsy-jq-evaluate)
- Fixed the injections in config.yaml to be wrapped with ""
```